lib/matplotlib/gridspec.py

```diff
@@ -57,39 +66,85 @@ def get_geometry(self):
         return self._nrows, self._ncols
 
     def get_subplot_params(self, figure=None):
+        # Must be implemented in subclasses
         pass
 
     def new_subplotspec(self, loc, rowspan=1, colspan=1):
-        """Create and return a `.SubplotSpec` instance."""
+        """
+        Create and return a `.SubplotSpec` instance.
+
+        Parameters
+        ----------
+        loc : (int, int)
+            A tuple ``(row_index, column_index)`` defining the position of the
```

Comment:
Minor wording point: I tend to prefer something like "The position of the subplot in the grid, as a ``(row_index, column_index)`` pair(/tuple)." which avoids semantically empty words like "defining".